RealmSwift/Tests/ObjectTests.swift

```diff
@@ -715,6 +850,220 @@ class ObjectTests: TestCase {
         token.invalidate()
     }
 
+    // How could these expectations be broken down into their own tests?
```

Comment:
Could you just have the setup either in every test or extracted into a different function and then have each expectation block as a its own test?

Basically the same way we've done it with the backlink tests. They are split too.